test/Constraints/function_builder_diags.swift

```diff
@@ -638,7 +638,7 @@ struct MyView {
 
   @TupleBuilder var invalidCaseWithoutDot: some P {
     switch Optional.some(1) {
-    case none: 42 // expected-error {{cannot find 'none' in scope}}
+    case none: 42 // expected-error {{cannot find 'none' in scope}} expected-note {{add 'let' or 'var' keyword to bind 'none' in a pattern}}
```

Comment:
I'm not sure about this fix-it here is the best option ... given `Optional` enum has a `case none` IMO the user's intention is more likely to be `case .none: 42`  than `case let none: 42`. Do you think this makes sense?

Reply:
I agree that there is a better fix-it for this situation (i.e., the "smarter" fix-it in the PR description). But, the better fix-it requires more effort and my current focus is not on extensive diagnostic improvements.